### ISSUE TITLE
'gx view' not working on packages installed globally

### ISSUE THREAD

**Issue by Kubuxu** [whyrusleeping/gx#79]
which is default now.

Reproduce:
```
> gx install
> gx view go-libp2p gx.dvcsimport
ERROR: open /home/kubuxu/go/src/github.com/ipfs/go-ipfs/vendor/gx/ipfs/QmQkQP7WmeT9FRJDsEzAaGYDparttDiB6mCpVBrq2MuWQS/go-libp2p/package.json: no such file or directory
```
In go-ipfs repo. It tires use local installation path.

**Comment by whyrusleeping** [comment]
thanks for reporting!

**Comment by whyrusleeping** [comment]
This should be resolved now in 5183acc020553516efcf2bbffb9f4bc0b0110d92
